lib/engine/game/g_18_ga/game.rb

```diff
@@ -40,6 +40,16 @@ class Game < Game::Base
 
         GAME_END_CHECK = { bankrupt: :immediate, stock_market: :current_or, bank: :current_or }.freeze
 
+        OPTION_REMOVE_HEXES = ['G13'].freeze
```

Comment:
Minor, but there are other options in 18GA that removes tiles (rule 7d) so consider including COTTON or COTTON_PORT in the game. Same of OPTION_ADD_HEXES.

I suppose a future implementation can do the renaming if needed, but maybe then a comment before line 43 that these two belong to the Cotton Port variant?

Reply:
For what it's worth, rule 7d removes tiles. These are map hexes that are being removed. No other variant does that.

Reply:
Done anyway.